src/pages/Main/components/ContentHeader/index.tsx

```diff
@@ -1,39 +1,21 @@
-import { useNavigate, useParams } from 'react-router-dom';
+import { useLocation, useNavigate, useParams } from 'react-router-dom';
 import { Button, Text } from '@/components';
-import { useSelectedChannel } from '@/hooks/useSelectedChannel';
-import { useSelectedStatus } from '@/hooks/useSelectedStatus';
 import { ContentHeaderWrapper } from '@/pages/Main/components/ContentHeader/style';
 
 const ContentHeader = () => {
   const { channelId } = useParams();
   const navigate = useNavigate();
+  const channelName = useLocation().state as string;
```

Comment:
```suggestion
  const {state} = useLocations() as {state : string}
```

이게 나을 것 같네요 !

그리고 사실 state가 없을때도 감안해야 합니다. 

```suggestion
  const {state} = useLocations()
  if (!state) return <Navaigate> 
```

이런식으로요 !

Reply:
오호 그렇군요!, 추가적으로, 나중에 error boundary 작업을 한다 가정하고
 `if (!state) throw new Error('errer');` 이런 식으로 일단 수정하는것이 나을까요?

Reply:
생각엔 404페이지로 라우팅하는게 더 적절하지 않을까 싶긴합니다 !